app/helpers/forever_style_guide/application_helper.rb

```diff
@@ -3,6 +3,315 @@
 module ForeverStyleGuide
   module ApplicationHelper
 
+    #Path helpers for mounted style guide use
+    def www_url(path = '/', url = nil)
+      strip_subdomain("www", path, url)
+    end
+
+    def web_app_url(path = '/', url = nil)
+      strip_subdomain("my", path, url)
+    end
+
+    def store_url(path = '/', url = nil)
+      strip_subdomain("store", path, url)
+    end
+
+    def strip_subdomain(sub, path = '/', url = nil)
+      url ||= request.url if respond_to?(:request)
+      url = URI(url)
+      replace = url.to_s
+      protocols = ['www.', 'store.', 'my.']
+      replace = protocols.find { |protocol| replace.include?(protocol) }
+
+      #protect against a nil return that would break localhost
+      if replace.blank?
+        replace ='/'
+      end
+
+      url.host = url.host.sub(replace, "#{sub}.")
+      url.path = path
+      url.query = nil
+      url.to_s
+    end
+
+    # Web App Paths
+    def login_path
```

Comment:
All of the _path helpers should probably be _url instead.  _path is generally used only for relative routes.